### PR TITLE
Config for olivier

### PR DESCRIPTION
**Pull request by teolemon**

### CODE REVIEW

File: docker/backend-dev/conf/Config2.pm [hunk 1]
```diff
@@ -61,18 +76,29 @@ $mongodb = "off";
 $mongodb_host = "mongodb://mongodb:27017";
 $mongodb_timeout_ms = 50000; # config option max_time_ms/maxTimeMS
 
-$memd_servers = [ "memcached:11211" ];
+$memd_servers = [ "127.0.0.1:11211" ];
```
Comment (hangy) [inline docker/backend-dev/conf/Config2.pm:79]: Not correct for all dev containers (local development)

**Comment by hangy** [comment]
Have you tested the local development still works?

**Comment by teolemon** [comment]
@hangy it's just to help @ocervell with debugging. Marking as WIP to reflect that

File: docker/backend-dev/conf/Config2.pm (hunk 1 above)
Comment (ocervell) [reply docker/backend-dev/conf/Config2.pm:79]: It's fine to keep the existing `memcached:11211` here, since the containers are running on the same VM.

**Comment by ocervell** [comment]
Thanks, closing now :)

**Pull request closed without merging**